src/config.rs

```diff
@@ -211,7 +316,7 @@ impl TryFrom<VariantDictionary> for KdfSettings {
     }
 }
 
-#[derive(Debug)]
+#[derive(PartialEq, Debug)]
```

Comment:
`PartialEq` might not be needed

Reply:
Addressed in https://github.com/sseemayer/keepass-rs/pull/85